ESPConnector/pom.xml

```diff
@@ -3,11 +3,11 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>inc.softserve</groupId>
-  <artifactId>SendEmailService</artifactId>
+  <artifactId>ESPConnector</artifactId>
   <version>1.0-SNAPSHOT</version>

```

Comment:
I believe you can avoid separate version here. As well as in referencing other modules.